### DISCUSSION TITLE
Uses "Flow Nord" theme, overlaps with text

### DISCUSSION THREAD

**Discussion by kerim** [YU000jp/logseq-plugin-bullet-point-custom-icon#3] [Q&A] [answered]
<img width="257" alt="Screenshot 2023-10-16 at 8 24 40 PM" src="https://github.com/YU000jp/logseq-plugin-bullet-point-custom-icon/assets/59976/cdaeb626-8f71-4ad5-82e7-22f38c763c83">

Thanks for making this, but somehow the icons appear so large that they overlap with the text. Perhaps it is my custom theme, but I don't know how to fix it?

**Comment by YU000jp** [comment]
The setting item exists in the plugin settings. ("Large icon?")
![image](https://github.com/YU000jp/logseq-plugin-bullet-point-custom-icon/assets/111847207/6576c1cc-0374-4630-bf2c-66b07dbbced4)

However, if it's influenced by other Custom CSS or themes, will need to look into that. Try setting it to standard and see what affects it.

**Comment by kerim** [comment]
It is a little better if I set it to "small" but on further inspection, the real problem seems to be the space between the icon and the start of the text, not the icon size. 

<img width="260" alt="Screenshot 2023-10-16 at 9 03 48 PM" src="https://github.com/YU000jp/logseq-plugin-bullet-point-custom-icon/assets/59976/5d7b3515-35ea-45cd-a933-ff7a9b381907">

**Comment by YU000jp** [comment]
It is most likely influenced by Custom CSS. Without looking at your CSS code, I can't see what CSS is being applied.

**Comment by kerim** [comment]
I'm using the Flow Nord theme (with a few modifications, but mostly for color):

https://github.com/henices/logseq-flow-nord

**Comment by YU000jp** [comment]
I enabled the theme, but it didn't look like that. I can't reproduce it in my environment.
Try disabling Custom CSS temporarily and apply partially. Need to find out which CSS code is causing the problem.

**Comment by kerim** [comment]
Seems to be the custom code I have to make the font size bigger:

`  --cl-font-size: 1.15em;`

But I need that to make the text readable...

**Comment by YU000jp** [comment]
In my environment, that alone does not cause the icon and text to overlap. It seems like there are other things as well. Alternatively, release it and press [Ctrl +] or [Ctrl -] to zoom in or out.

**Comment by YU000jp** [comment]
After that....

I found the code that is probably the cause. From the theme. https://github.com/henices/logseq-flow-nord/blob/5cbb665d823d8994182d5ed1da2c1bd5aebefff3/src/code/interface.css#L376


Applying the following CSS code may solve the problem.
```
main div.block-main-container div.bullet-container div.bullet {
    font-size: unset;
}
```

**Comment by kerim** [comment, marked as answer]
Thank you so much. I appreciate you taking the time to look into this for me. I modified the code a little, but this works:

```
.block-main-container .bullet-container .bullet {
    font-size: unset !important;
}
```
